capstone-backend/routes/callback.js

```diff
@@ -0,0 +1,37 @@
+var express = require('express');
+var router = express.Router();
+const request = require('superagent');
+require('dotenv').config()
+
+/* Handle LinkedIn OAuth callback and return user profile. */
+router.get('/', function(req, res, next) {
```

Comment:
We can change the code in this function to use async await, instead of raw promises.